site/content/docs/5.3/components/accordion.md

```diff
@@ -22,6 +22,8 @@ The accordion uses [collapse]({{< docsref "/components/collapse" >}}) internally
 Click the accordions below to expand/collapse the accordion content.
 
 {{< example >}}
+
+```html
```

Comment:
This modification totally breaks the layout and is not needed (see https://deploy-preview-38147--twbs-bootstrap.netlify.app/docs/5.3/components/accordion/#example).